website/docs/components/form/select/partials/guidelines/guidelines.md

```diff
@@ -1,3 +1,11 @@
+!!! Info
+
+**This component uses a native HTML element**
+
+It's not possible to create custom variations of this component as it uses a native HTML element. Find more information [on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select).
```

Comment:
Should we be more specific, and explain that it's not possible to create custom variations of the `<option>` elements?

Reply:
Isn't this also applicable to the `<select>` element too, where we can't apply non-native HTML customizations to it?

Reply:
Initially when updating this my intent was to make it seem that any "custom variation" (which to your point Jory may be not accurate enough) within the select component (meant to be technical variation) was not possible and intentionally kept it vague so that it encompasses more than just the `<option>`.

But to your point, maybe we need to say "any technical custom variation" isn't allowed? I Want to include more than the option list so that this is "future" proof and that no one comes and says we didn't explicitly say the `<select>` element wasn't mentioned.

Reply:
Ok wording was updated again.